src/main/java/core/basesyntax/MainApp.java

```diff
@@ -13,4 +13,15 @@
  */
 public class MainApp {
 
+    public static void main(String[] args) {
+        Track track = new Track();
```

Comment:
What will happen if we write:
Machine excavator = new Excavator();
and the same with other objects?

Why do you think we create abstract class Machine?